### PR TITLE
For review: Add variable rate slope 1 stablecoin update.

### PR DESCRIPTION
**Pull request by reem**
Same issue with the Metis portion as #1

### CODE REVIEW

**Review by katiebk — CHANGES_REQUESTED** [review]
amending my previous comment

all USDC going from 0.035 -> 0.05 EXCEPT:
- v3 Base USDbC 0.04 -> 0.05
- v2 Avalanche USDC 0.04 -> 0.05
- v2 Ethereum USDC 0.04 -> 0.05
- v2 Polygon USDC 0.04 -> 0.05

File: diffs/AaveV2Avalanche_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV2Avalanche_IncreaseStablecoinOptimalBorrowRates_20231116_after.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+## Reserve changes
+
+### Reserve altered
+
+#### USDC.e ([0xA7D7079b0FEaD91F3e65f86E8915Cb59c1a4C664](https://snowtrace.io/address/0xA7D7079b0FEaD91F3e65f86E8915Cb59c1a4C664))
+
+| description | value before | value after |
+| --- | --- | --- |
+| interestRateStrategy | [0xD96B68638bdbb625A49F5BAC0dC3B66764569d30](https://snowtrace.io/address/0xD96B68638bdbb625A49F5BAC0dC3B66764569d30) | [0x116EFD5652A9993A5984055B2da7eb9acfB48Fd2](https://snowtrace.io/address/0x116EFD5652A9993A5984055B2da7eb9acfB48Fd2) |
+| variableRateSlope1 | 4 % | 5 % |
```
Comment (katiebk) [inline diffs/AaveV2Avalanche_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV2Avalanche_IncreaseStablecoinOptimalBorrowRates_20231116_after.md:10]: 3.5% ->  5%

File: diffs/AaveV2Ethereum_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV2Ethereum_IncreaseStablecoinOptimalBorrowRates_20231116_after.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+| interestRateStrategy | [0xaC63290BC16fBc33353b14f139cEf1c660ba56F0](https://etherscan.io/address/0xaC63290BC16fBc33353b14f139cEf1c660ba56F0) | [0xd39D46597f78F220d17Ef0811574aE71810abd83](https://etherscan.io/address/0xd39D46597f78F220d17Ef0811574aE71810abd83) |
+| variableRateSlope1 | 4 % | 5 % |
+| interestRate | ![before](/.assets/e6957fe4c310ba51f499314c131f0977304a09d1.svg) | ![after](/.assets/1661f76a7e96cd3c1d406fdc47aed4a08f1c5ddd.svg) |
+
+#### USDC ([0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48](https://etherscan.io/address/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48))
+
+| description | value before | value after |
+| --- | --- | --- |
+| interestRateStrategy | [0x8Cae0596bC1eD42dc3F04c4506cfe442b3E74e27](https://etherscan.io/address/0x8Cae0596bC1eD42dc3F04c4506cfe442b3E74e27) | [0x312520255B081ED3f9C567a4bb9348A3327a1061](https://etherscan.io/address/0x312520255B081ED3f9C567a4bb9348A3327a1061) |
+| variableRateSlope1 | 4 % | 5 % |
```
Comment (katiebk) [inline diffs/AaveV2Ethereum_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV2Ethereum_IncreaseStablecoinOptimalBorrowRates_20231116_after.md:50]: 3.5% -> 5%

File: diffs/AaveV2Polygon_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV2Polygon_IncreaseStablecoinOptimalBorrowRates_20231116_after.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+## Reserve changes
+
+### Reserve altered
+
+#### USDC ([0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174](https://polygonscan.com/address/0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174))
+
+| description | value before | value after |
+| --- | --- | --- |
+| interestRateStrategy | [0xCE5870016D3cFa4e1c71Fb7c7EE8Cef67712a7a3](https://polygonscan.com/address/0xCE5870016D3cFa4e1c71Fb7c7EE8Cef67712a7a3) | [0x283Df7893eF10F729890017F57d76B8D78e18915](https://polygonscan.com/address/0x283Df7893eF10F729890017F57d76B8D78e18915) |
+| variableRateSlope1 | 4 % | 5 % |
```
Comment (katiebk) [inline diffs/AaveV2Polygon_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV2Polygon_IncreaseStablecoinOptimalBorrowRates_20231116_after.md:10]: 3.5% -> 5%

File: diffs/AaveV3Base_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV3Base_IncreaseStablecoinOptimalBorrowRates_20231116_after.md
Comment (katiebk) [inline diffs/AaveV3Base_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV3Base_IncreaseStablecoinOptimalBorrowRates_20231116_after.md:1]: base isn't included in the hackmd but the snapshot is for all markets. variableslope1=0.04 for USDbC so this is correct

File: diffs/AaveV3Metis_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV3Metis_IncreaseStablecoinOptimalBorrowRates_20231116_after.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+## Reserve changes
+
+### Reserve altered
+
+#### m.USDC ([0xEA32A96608495e54156Ae48931A7c20f0dcc1a21](https://andromeda-explorer.metis.io/address/0xEA32A96608495e54156Ae48931A7c20f0dcc1a21))
+
+| description | value before | value after |
+| --- | --- | --- |
+| interestRateStrategy | [0x3B88f83B2b0444F14B6fc904040F648206F551fc](https://andromeda-explorer.metis.io/address/0x3B88f83B2b0444F14B6fc904040F648206F551fc) | [0x3c8DF161A59F2a3ed4D40F073395d6a0f14290C0](https://andromeda-explorer.metis.io/address/0x3c8DF161A59F2a3ed4D40F073395d6a0f14290C0) |
+| variableRateSlope1 | 4 % | 5 % |
```
Comment (katiebk) [inline diffs/AaveV3Metis_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV3Metis_IncreaseStablecoinOptimalBorrowRates_20231116_after.md:10]: we don't have interest rate parameters ingested for metis so i can't use bigquery to confirm whether the before value is correct

**Review by katiebk — APPROVED** [review]

File: src/20231116_Multi_IncreaseStablecoinOptimalBorrowRates/IncreaseStablecoinOptimalBorrowRates.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+## Summary
+
+This AIP proposes to make adjustments to stablecoin interest rate models across Aave v2 and v3 markets. Specifically, this AIP proposes to increase the slope 1 parameter for stablecoins to implement a 5% optimal borrow rate.
+
+## Specification
+
+| Market       | Asset  | Parameter          | Current Value | Recommended Value |
+| ------------ | ------ | ------------------ | ------------- | ----------------- |
+| Ethereum v2  | USDC   | variableRateSlope1 | 0.035         | 0.05              |
```
Comment (katiebk) [inline src/20231116_Multi_IncreaseStablecoinOptimalBorrowRates/IncreaseStablecoinOptimalBorrowRates.md:15]: so this actually needs to be changed then for the hackmd. all v2 USDC current values = 0.04

File: src/20231116_Multi_IncreaseStablecoinOptimalBorrowRates/IncreaseStablecoinOptimalBorrowRates.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+| Ethereum v2  | FRAX   | variableRateSlope1 | 0.04          | 0.05              |
+| Ethereum v2  | sUSD   | variableRateSlope1 | 0.04          | 0.05              |
+| Ethereum v2  | GUSD   | variableRateSlope1 | 0.04          | 0.05              |
+| Ethereum v2  | LUSD   | variableRateSlope1 | 0.04          | 0.05              |
+| Ethereum v2  | USDP   | variableRateSlope1 | 0.04          | 0.05              |
+| Ethereum v3  | USDC   | variableRateSlope1 | 0.035         | 0.05              |
+| Ethereum v3  | USDT   | variableRateSlope1 | 0.04          | 0.05              |
+| Ethereum v3  | LUSD   | variableRateSlope1 | 0.04          | 0.05              |
+| Ethereum v3  | FRAX   | variableRateSlope1 | 0.04          | 0.05              |
+| Avalanche v2 | USDC   | variableRateSlope1 | 0.035         | 0.05              |
```
Comment (katiebk) [inline src/20231116_Multi_IncreaseStablecoinOptimalBorrowRates/IncreaseStablecoinOptimalBorrowRates.md:26]: change here

File: src/20231116_Multi_IncreaseStablecoinOptimalBorrowRates/IncreaseStablecoinOptimalBorrowRates.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+| Avalanche v2 | USDT   | variableRateSlope1 | 0.04          | 0.05              |
+| Avalanche v2 | DAI    | variableRateSlope1 | 0.04          | 0.05              |
+| Avalanche v3 | USDC   | variableRateSlope1 | 0.035         | 0.05              |
+| Avalanche v3 | USDT   | variableRateSlope1 | 0.04          | 0.05              |
+| Avalanche v3 | DAI    | variableRateSlope1 | 0.04          | 0.05              |
+| Avalanche v3 | FRAX   | variableRateSlope1 | 0.04          | 0.05              |
+| Polygon v2   | USDC   | variableRateSlope1 | 0.035         | 0.05              |
```
Comment (katiebk) [inline src/20231116_Multi_IncreaseStablecoinOptimalBorrowRates/IncreaseStablecoinOptimalBorrowRates.md:33]: change here

**Review by katiebk — CHANGES_REQUESTED** [review]

**Comment by reem** [comment]
Updated current values in proposal description

**Review by katiebk — APPROVED** [review]

**Review by ndelzingaro88 — COMMENTED** [review]
Added some comments for the PR

File: diffs/AaveV3Arbitrum_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV3Arbitrum_IncreaseStablecoinOptimalBorrowRates_20231116_after.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
+## Reserve changes
```
[the remark this answers is not in the document]
Comment (reem) [reply diffs/AaveV3Arbitrum_IncreaseStablecoinOptimalBorrowRates_20231116_before_AaveV3Arbitrum_IncreaseStablecoinOptimalBorrowRates_20231116_after.md:1]: Walter removed MAI, context I think is the MAI deprecation proposal also going up on monday

**Pull request closed without merging**